save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -38,14 +37,16 @@ class FixPlugin(testConfig: TestConfig, testFiles: List<String> = emptyList()) :
         .build()
 
     override fun handleFiles(files: Sequence<List<Path>>): Sequence<TestResult> {
-        val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
-        val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().singleOrNull()
         val flattenedResources = files.toList()
         if (flattenedResources.isEmpty()) {
```

Comment:
```suggestion
        if (files.any { it.isNotEmpty() }) {
```
we don't need the whole sequence here

Reply:
Agree, for `if` we don't need the whole sequence, however we use it for `logInfo`